### PR TITLE
fix: option to filter events that started by Screwdriver scheduler

### PR DESCRIPTION
**Pull request by adong**
## Context

Follow up of https://github.com/screwdriver-cd/ui/pull/914

Sample Pipeline: https://cd.screwdriver.cd/pipelines/11326/events/735254

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by jithine — APPROVED** [review]
are there no tests to update ?

File: app/components/pipeline-events/component.js [hunk 1]
```diff
@@ -378,20 +381,27 @@ export default Component.extend(ModelReloaderMixin, {
...
             (event, idx) => event.status !== 'SKIPPED' || idx === 0
           );
+        }
 
-          return filteredEvents;
+        // filter events created by screwdriver scheduler
+        if (this.filterSchedulerEvents) {
+          filteredEvents = filteredEvents.filter(event => {
+            return event.creator.name !== SD_SCHEDULER;
+          });
```
Comment (tkyi) [inline app/components/pipeline-events/component.js:401]:
```suggestion
          filteredEvents = filteredEvents.filter(event => event.creator.name !== SD_SCHEDULER);
```

File: app/components/pipeline-options/component.js [hunk 2]
```diff
@@ -357,6 +370,9 @@ export default Component.extend({
         await this.shuttle.updatePipelineSettings(pipeline.id, {
           filterSchedulerEvents
         });
+        this.set('successMessage', `Pipeline preferences updated successfully`);
```
Comment (tkyi) [inline app/components/pipeline-options/component.js:373]: 👍

**Review by tkyi — DISMISSED** [review]
Small suggestion otherwise lgtm

**Review by tkyi — APPROVED** [review]

**Pull request merged**